email_automation/email-config.py

```diff
@@ -116,12 +137,6 @@ def update_user_pool(pool_id, pool_name, html, identity_arn, cognito_client):
   response = cognito_client.update_user_pool(
         UserPoolId= pool_id,
         AutoVerifiedAttributes=['email'],
-
-        MfaConfiguration='ON',
-        DeviceConfiguration={
-            'ChallengeRequiredOnNewDevice': True,
-            'DeviceOnlyRememberedOnUserPrompt': True
-        },
```

Comment:
why is this removed?

Reply:
> why is this removed?

Jianli specifically asked me to remove it when we were going over the specifics of my usage of update_user_pool. These settings are already set the exact same way when the user pool is created, so there's no reason to add code to change it.